### ISSUE TITLE
Christmas Day is two times in Slovakia

### ISSUE THREAD

**Issue by mariusz-korzekwa** [novafloss/workalendar#205]
Hi again.

In slovakia, christmas day occure 2 times.

```
sk = Slovakia()
sk.holidays()
[
(datetime.date(2017, 1, 1), 'Day of the Establishment of the Slovak Republic'), 
(datetime.date(2017, 1, 1), 'New year'), 
(datetime.date(2017, 1, 6), 'Epiphany'), 
(datetime.date(2017, 4, 14), 'Good Friday'), 
(datetime.date(2017, 4, 17), 'Easter Monday'), 
(datetime.date(2017, 5, 1), 'Labour Day'), 
(datetime.date(2017, 5, 8), 'Liberation Day'), 
(datetime.date(2017, 7, 5), 'Saints Cyril and Methodius Day'), 
(datetime.date(2017, 8, 29), 'Slovak National Uprising anniversary'), 
(datetime.date(2017, 9, 1), 'Day of the Constitution of the Slovak Republic'),
(datetime.date(2017, 9, 15), 'Day of Blessed Virgin Mary, patron saint of Slovakia'), (datetime.date(2017, 11, 1), 'All Saints\xe2\x80\x99 Day'), 
(datetime.date(2017, 11, 17), 'Struggle for Freedom and Democracy Day'), 
(datetime.date(2017, 12, 24), 'Christmas Eve'), 
(datetime.date(2017, 12, 25), 'Christmas Day'), 
(datetime.date(2017, 12, 25), 'Christmas Day'), 
(datetime.date(2017, 12, 26), "St. Stephen's Day (The Second Christmas Day)")
]
```
I think is because of:
In `ChristianMixin` you have `include_christmas = True`
and in `Slovakia` class you have `(12, 25, "Christmas Day"),`

**Comment by brunobord** [comment]
oh, yes indeed. could you please submit a PR to fix this?

**Comment by brunobord** [comment]
never mind, I took half an hour to solve this.

**Comment by brunobord** [comment]
[fixed and available in the v1.3.0 release](https://github.com/novafloss/workalendar/releases/tag/1.3.0)
